conans/test/integration/command/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:22.04
```

Comment:
The 3 dockerfiles are almost identical, isn't it possible to layer them so testing with them is cheaper (faster images to build)?